adapter/internal/oasparser/model/api_yaml.go

```diff
@@ -110,11 +110,11 @@ type PolicyList []Policy
 
 // Policy holds APIM policies
 type Policy struct {
-	PolicyName    string      `json:"policyName,omitempty"`
-	PolicyVersion string      `json:"policyVersion,omitempty"`
-	Action        string      `json:"-"`
-	Order         int         `json:"order,omitempty"`
-	Parameters    interface{} `json:"parameters,omitempty"`
+	PolicyName       string      `json:"policyName,omitempty"`
+	PolicyVersion    string      `json:"policyVersion,omitempty"`
+	Action           string      `json:"-"` // This is a meta value used in CC, not included in API YAML
+	IsPassToEnforcer bool        `json:"-"` // This is a meta value used in CC, not included in API YAML
```

Comment:
What does this mean?

Reply:
This `action` and `IsPassToEnforcer ` is not coming from the API YAML file (but others we get from it). We include these values in the policy for the internal process of the policy

Reply:
What would happen if `IsPassToEnforcer` was not set to true?

Reply:
If it is not true, then the policy is not passed to the enforcer. That means the policy is not supported by the enforcer. For example, Call Interceptor Service API Policy do not want to be passed to the enforcer.